### PR TITLE
create react app with all features

### PR DESCRIPTION
**Pull request by sireclifford**
**Home page** 

- When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.

- Along with the category name, you will display some numeric values per category that come from the API.

- When the user clicks (or taps) on a category item, the application navigates to the details page.

**Details page**

- On the details page, the web app retrieves data from the API to show detailed data in the category.

- When the user clicks on the "Back" button (<), the user navigates to the home page.

**Screenshot**
metric-app-full

**Test Shot**
metric-app-test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,144 @@
...
+  </a>
 
-[Live Demo Link](https://livedemo.com)
+<h3 align="center">Metrics App</h3>
 
+  <p align="center">
+    It's about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+    <br />
+    <br />
+    <a href="https://github.com/github_username/repo_name">View Demo</a>
```
Comment (zhadier) [inline README.md]: - [x] Please ensure that you deploy the Project using Heroku, Netlify or GitHub Pages and everything is working perfectly in the deployment. Kindly also add the deployment link in the ReadMe as is a Requirement for this project.

File: src/components/genre/Genre.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    navigate(`/details/${genre}`);
+  };
+
+  useEffect(() => {
+    if (status === 'idle') {
+      dispatch(fetchAllMovies());
+    }
+  }, [dispatch, status]);
+
+  let content;
```
Comment (zhadier) [inline src/components/genre/Genre.js]: - [x] Kindly ensure that the home page, which is the genre in your case, has a list of categories that could be filtered using given parameters, e.g. genre name. You can use a search bar to implement the filter.

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "Oxygen",
+    "Ubuntu",
+    "Cantarell",
+    "Fira Sans",
+    "Droid Sans",
+    "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #ec4b8a;
```
Comment (zhadier) [inline src/index.css]: - [x] Kindly ensure that you use the same typography as provided by the Behance design for all of your components. I've added an image below for your ease.

image

File: src/_test_/movie.test.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    title: 'test',
+    rating: '1',
+  };
+  const tree = renderer.create(
+    <Provider store={store}>
+      <Movie movie={movie} />
+    </Provider>,
+  );
+  expect(tree).toMatchSnapshot();
+});
```
Comment (zhadier) [inline src/_test_/movie.test.js:19]: - [x] Kindly ensure that you add unit tests for all of your reducers. Reducers are pure functions by default so you can quite easily test them if you provide a stub action. Please ensure that you add unit tests for all of your reducers.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @sireclifford  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ GitFlow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️

- [x] Kindly ensure that you recorded a video presenting the features of the project. It must conform to the following and the link should be added in the ReadMe and the PR.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, highlighting all the project features.
  - The video contains a highlight about a piece of the code that you find interesting or are proud of making.

- [x] Please ensure that your commit messages are descriptive from no onwards. There's no way to distinguish what you've implemented in a commit from your recent commit messages. Having descriptive commit messages will help you in the long run if you have to revert to an older version.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sireclifford) [reply README.md]: I has been done

File: src/components/genre/Genre.js (hunk 2 above)
Comment (sireclifford) [reply src/components/genre/Genre.js]: It has been modified

File: src/index.css (hunk 3 above)
Comment (sireclifford) [reply src/index.css]: It has been modified

File: src/_test_/movie.test.js (hunk 4 above)
Comment (sireclifford) [reply src/_test_/movie.test.js:19]: test for reducers has been added

**Pull request merged**